command/loadbot/loadbot_command.go

```diff
@@ -96,14 +96,16 @@ func (l *LoadbotCommand) Run(args []string) int {
 	var valueRaw string
 	var count uint64
 	var jsonrpc string
-
+	var maxConns int
+	defaultMaxConns := int(2 * tps)
```

Comment:
this will be zero because `tps` is not set yet

Reply:
My Bad, Fixed it commit [link](https://github.com/0xPolygon/polygon-sdk/pull/295/commits/22bf32d3e2e575e6582e0d2f2325e4fbfca478d3)

Reply:
@munna0908 Hi, please note the value is set by calling `flags.Parse(args)`.

Reply:
Fixed it commit-[link](https://github.com/0xPolygon/polygon-sdk/pull/295/commits/5511eda215fb0f798230c10ad91357a698ae37d7)